<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>deleteWindow</class>
 <widget class="QDialog" name="deleteWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>305</width>
    <height>102</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog {
background-color: rgb(27,27,27);
}

QMessageBox {
background -color: rgb(27,27,27);
color: #ffffff;
}

QMessageBox QLabel {
font: 600 10pt &quot;Yu Gothic UI Semibold&quot;;
color: rgb(255, 255, 255);
}

QMessageBox QPushButton::hover {
background-color: #666;
}

QMessageBox QPushButton::pressed {
background-color: #888;
}

QMessageBox QPushButton {
color: #ffffff;
background-color: #333;
font: 600 10pt &quot;Yu Gothic UI Semibold&quot;;
}</string>
  </property>
  <widget class="QRadioButton" name="radioButtonEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>151</width>
     <height>26</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
QRadioButton {
font: 600 10pt &quot;Yu Gothic UI Semibold&quot;;
color: rgb(255, 255, 255);
}

QRadioButton:checked + label::before {
  background: #9FD468;
}

QRadioButton:checked + label::after {
  left: 27px;
}


QRadioButton::after {
  content: '';
  position: absolute;
  top: 9px;
  left: 12px;
  width: 0px;
  height: 0px;
  border-radius: 50%;
  background: rgb(165, 165, 165);
  box-shadow: inset 0 1px 1px rgba(0,0,0,.5);
  opacity: 0;
  transition: .2s;
}

QRadioButton:checked + label::after {
  opacity: 1;
  top: 1px;
  left: 4px;
  width: 16px;
  height: 16px;
}

</string>
   </property>
   <property name="text">
    <string>Редактировать</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonCancel">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>70</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
color: #ffffff;
background-color: #333;
font: 600 10pt &quot;Yu Gothic UI Semibold&quot;;
}

QPushButton::hover {
background-color: #666;
}

QPushButton::pressed {
background-color: #888;
}</string>
   </property>
   <property name="text">
    <string>Отмена</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButtonBooking">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>151</width>
     <height>26</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 600 10pt &quot;Yu Gothic UI Semibold&quot;;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Забронировать</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButtonDelete">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>112</width>
     <height>26</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 600 10pt &quot;Yu Gothic UI Semibold&quot;;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Удалить</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonOk">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>70</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
color: #ffffff;
background-color: #333;
font: 600 10pt &quot;Yu Gothic UI Semibold&quot;;
}

QPushButton::hover {
background-color: #666;
}

QPushButton::pressed {
background-color: #888;
}</string>
   </property>
   <property name="text">
    <string>Ok</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
